Scripts/script-ContextSearchForString.yml

```diff
@@ -0,0 +1,51 @@
+commonfields:
+  id: ContextSearchForString
+  version: -1
+name: ContextSearchForString
+script: |+
+  var path = args.path;
+  var str = args.str;
+  var flat={};
+
+  function searchString(obj) {
+      flattenFields(obj,undefined,flat);
+      keysArr = Object.keys(flat);
+      for (i = 0; i < keysArr.length - 1; i++) {
+          if (flat[keysArr[i]] == str) {
```

Comment:
===

Reply:
read about differences between == and === in javascript 
https://stackoverflow.com/questions/523643/difference-between-and-in-javascript